fuse/namespace.cpp

```diff
@@ -241,38 +235,58 @@ bool create_file(const std::string& dest) {
 // the layered mounts.
 bool do_mounts_from_json(const JAST& jast, const std::string& fuse_mount_path)
 {
+	// Create a known mount-able dir in the parent mount namespace to base the rest of the mounts on.
+	const std::string mount_prefix = "/tmp/.wakebox-mount";
+	if (0 != mkdir(mount_prefix.c_str(), 0777)) {
```

Comment:
I wonder if we should perhaps only do the mkdir if there is a destination=="/".

An easy way to achieve this would be:
- eliminate replacedRoot
- leave mount_prefix initially unset (ie: "")
- always prepend mount_prefix to target
- whenever you find a destination=="/", run `mount_prefix += "/tmp/.wakebox-mount" && mkdir(mount_prefix)`
- if (!mount_prefix.empty()) run pivot_root

This even makes the tool behave somewhat sensibly if there are two mounts at destination=="/".